src/averageAges.js

```diff
@@ -37,7 +41,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const woman = people.filter(person => withChildren !== undefined
+    ? person.sex === 'f' && people.find(human => person.name === human.mother)
+    : person.sex === 'f');
+
```

Comment:
```suggestion
  const woman = people.filter(person => withChildren
    ? people.find(human => person.name === human.mother)
    : person.sex === 'f');

```

те що вона мама - робить з неї жінку. Мабуть...)